docs/sdks/uid2-sdk-ref-android.md

```diff
@@ -0,0 +1,187 @@
+---
+title: UID2 SDK for Android
+description: Reference information about the Android SDK.
+hide_table_of_contents: false
+sidebar_position: 14
+---
+
+# UID2 SDK for Android Reference Guide
+
+<!-- This guide includes the following information:
+
+- [Functionality](#functionality)
+- [SDK Version](#sdk-version)
+- [Features](#features)
+- [GitHub Open-Source Repository](#github-open-source-repository)
+- [Minimum Requirements](#minimum-requirements)
+- [Installation](#installation)
+  -  [Installing with Gradle ](#installing-with-gradle)
+  -  [Installing with Maven ](#installing-with-maven)
+- [How to Use the SDK](#how-to-use-the-sdk)
+- [UID2Manager API](#uid2manager-api)
+  -  [Functions](#functions)
+  -  [Variables](#variables)
+- [Android Initialization](#android-initialization)
+- [Code Samples](#code-samples) -->
+
+You can use the UID2 SDK for Android to facilitate the process of establishing client identity using UID2 and retrieving advertising tokens on Android devices.
+
+## Functionality
+
+This SDK simplifies integration with UID2 for any publishers who want to support UID2 for apps running on Android devices. The following table shows the functions it supports.
+
+| Encrypt Raw UID2 to UID2 Token | Decrypt UID2 Token | Generate UID2 Token from DII | Refresh UID2 Token |
+| :--- | :--- | :--- | :--- |
+| No | No | No | Yes |
+
+## SDK Version
+
+<!-- As of 2023-07-15 -->
+
+This documentation is for the UID2 Android SDK version 0.4.0.
```

Comment:
can we also say As of 2023-08-01 on the page to the reader? and remove the comment line above?

If we mention version number might as well mention the date it's updated from

Reply:
As above. I'll treat them both the same per your response.